src/config/countries.php

```diff
@@ -933,7 +933,7 @@
     ],
     'UM' => [
         'name' => 'United States Outlying Islands',
-        'diallingCode' => ,
+        'diallingCode' => null,
```

Comment:
Take it you've checked them all?

Reply:
@mikefrancis Yep I have been through them otherwise